docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_9.md

```diff
@@ -0,0 +1,126 @@
+# ExperimentalDetectronGenerateProposalsSingleImage {#openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_9}
+
+**Versioned name**: *ExperimentalDetectronGenerateProposalsSingleImage-9*
+
+**Category**: *Object detection*
+
+**Short description**: The *ExperimentalDetectronGenerateProposalsSingleImage* operation computes ROIs and their scores
+based on input data.
+
+**Detailed description**: The operation performs the following steps:
+
+1.  Transposes and reshapes predicted bounding boxes deltas, variances and scores to get them into the same order as the
+anchors.
+2.  Transforms anchors into proposals using deltas, variances and clips proposals to an image.
+3.  Sorts all `(proposal, score)` pairs by score from highest to lowest; order of pairs with equal scores is undefined.
+4.  Takes top *pre_nms_count* proposals, if total number of proposals is less than *pre_nms_count* takes all proposals.
+5.  Removes predicted boxes with either height or width < *min_size*.
+6.  Excute nms operation and takes and returns top proposals. The max number of returned proposals is *post_nms_count*.
```

Comment:
also, please add description in what step `coordinates_offset` is applied and how?